data/json/items/gun/308.json

```diff
@@ -446,5 +455,51 @@
     "min_cycle_recoil": 2700,
     "pocket_data": [ { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "ar10mag_20rd", "ar10_makeshiftmag" ] } ],
     "melee_damage": { "bash": 12 }
+  },
+  {
+    "id": "rfb_308",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "7.62x51mm bullpup rifle" },
+    "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines of the heavy automatic rifle and a 7.62x51mm cartridge.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "rfb_308",
+        "name": { "str_sp": "Kel-Tec RFB" },
+        "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using magazines from the FN FAL and the .308 Winchester cartridge.  The main design features include forward case ejection, the duplication of the magazine release button and safety catch for left-handers, and a posted barrel for increased accuracy.",
```

Comment:
RFB stands for Rifle. Forward ejecting, bullpup, in case you wanted to mention that here.

I am not sure what you're referring to for the posted barrel bit. The RFB does not take any special approaches for accuracy, and the non-free floated nature of the barrel is regarded as a compromise.

Reply:
>  non-free floated nature of the barrel is regarded as a compromise.

Rechecked dissasemble, yes. It's somewhat resemble free-floating, but not fully. On other side, imo worth mentioning - barrel itsels is thicker and heavier, than FN FAL one, and used as "spine" for rifle. And, there is not so much plastic and aluminum, so, i think durability 6 is too low.

Descrption updated, durability changed to 7 from 6.